bin/projects.py

```diff
@@ -80,10 +80,7 @@ def __lt__(self, other: "Project") -> bool:
 
     @classmethod
     def header(cls) -> str:
-        return (
-            f"| {'Name':{cls.NAME}} | CI | OS | Notes |\n"
-            f"|{'':-^{cls.NAME+2  }}|----|----|:------|"
-        )
+        return f"| {'Name':{cls.NAME}} | CI | OS | Notes |\n" f"|{'':-^{cls.NAME+2  }}|----|----|:------|"
```

Comment:
There's a reason there's two lines, here: the actual string is also two lines long.

Reply:
Python's merging of strings that follow each other is _highly_ prone to mistakes. I last caught one five days ago https://github.com/scikit-hep/decaylanguage/pull/127#discussion_r600857204

I think if you added a `+`, this would format correctly. Or you could add a fmt on/off to clarify you know what you are doing.

Reply:
That's tricky though, yes. But there's still a difference with the issue you refer to.

> Or you could add a fmt on/off to clarify you know what you are doing.

I still don't like that, though. It distracts from what's actually going on, to me.